test/example/module-splitting.txt

```diff
@@ -283,10 +295,13 @@ Before:
 Keeping: <none>
 After:
 (module
+ (table $0 0 funcref)
+ (export "%table" (table $0))
```

Comment:
Why are we adding tables to every wasm-split outputs even when they are not necessary?

Reply:
At first I was a bit confused with the module-splitter, as it relies on the existence of the table object from the very beginning. So I just gave it the table for now. But I will try to fix this, so it only adds a table to the module when it needs to.

Reply:
Ok I was able to fix this, although it got hacky. Totally open to better suggestions.